app/controllers/cms_controller.rb

```diff
@@ -36,11 +22,51 @@ def clear_page_cache
     redirect_to request.fullpath.sub(%r{/refresh$}, "")
   end
 
+  def privacy
+    process_resource Cms::Pages::PrivacyNotice
+  end
+
+  def deep_test
+    process_resource Cms::Pages::DeepTest
+  end
+
   private
 
+  def process_collection(cls, title: "News & Updates", page_name: "Articles", collection_wrapper: "ncce-news-archive")
+    page =
+      if params[:page].present?
+        params[:page].to_i
+      else
+        1
+      end
```

Comment:
I don't think I picked up on this last time, but are we doing anything to ensure the page is inside a sensible range?

IE, what if I change the param to -1 or 100000?